layouts/partials/page-meta-links.html

```diff
@@ -0,0 +1,41 @@
+/* template adapted from Docsy theme */
```

Comment:
@sftim, This comment is displaying:
https://deploy-preview-27393--kubernetes-io-master-staging.netlify.app/docs/home/

Reply:
Ah, that's wrong. I didn't see it locally - maybe I didn't check properly.

Reply:
I feel this merits a :facepalm:!